node/external/transactionAPI/gasUsedAndFeeProcessor.go

```diff
@@ -0,0 +1,69 @@
+package transactionAPI
+
+import (
+	"math/big"
+
+	"github.com/multiversx/mx-chain-core-go/core"
+	"github.com/multiversx/mx-chain-core-go/data/transaction"
+)
+
+type gasUsedAndFeeProcessor struct {
+	txFeeCalculator feeComputer
```

Comment:
Calculator & Computer ? Let's rename to even-out the naming

Reply:
changed